build/package/package.sh

```diff
@@ -27,7 +27,7 @@ export VERSION_GIT
 
 echo "$GITHUB_APP_PEM" > app.pem
 GITHUB_TOKEN="$(go run ./tools/github-token -pem app.pem -app_id "$GITHUB_APP_ID" -owner 'mongodb' -repo 'mongodb-atlas-cli')"
-
+rm -rf app.pem
```

Comment:
rather than this worht adding to the gitignore

Reply:
I was thinking about it maybe a follow-up, deleting is more secure though